docs/developer-docs/latest/developer-resources/database-apis-reference/graphql-api.md

```diff
@@ -0,0 +1,476 @@
+
+Queries can accept a `filters` parameter with the following syntax:
+
+`filters: { field: { operator: value } }`
+
+Logical operators (`and`, `or`) can also be used and accept arrays of objects.
+
+The following operators are available:
+
+| Operator     | Description                      |
+| -------------| ---------------------------------|
+| `eq`         | Equal                            |
+| `ne`         | Not equal                        |
+| `lt`         | Less than                        |
+| `lte`        | Less than or equal to            |
+| `gt`         | Greater than                     |
+| `gte`        | Greater than or equal to         |
+| `in`         | Included in an array             |
+| `notIn`      | Not included in an array         |
+| `contains`   | Contains                         |
+| `ncontains`  | Does not contain                 |
+| `containsi`  | Contains, case sensitive         |
+| `ncontainsi` | Does not contain, case sensitive |
+| `null`       | Is null                          |
+| `notNull`    | Is null                          |
+| `between`    | Is between                       |
+| `startsWith` | Starts with                      |
+| `endsWith`   | Ends with                        |
+| `and`        | Logical `and`                    |
+| `or`         | Logical `or`                     |
+
+::: request Example query with filters
+
+```graphql
+{
+  documents(filters: { name: { eq: "test" }, or: [{ price: { gt: 10 }}, { title: { startsWith: "Book" }}] }) {
+    data {
+      id
+    }
+  }
+}
+```
+
+:::
+
+## Sorting
+
+Queries can accept a `sort` parameter with the following syntax:
+
+- to sort based on a single value: `sort: "value"` 
+- to sort based on multiple values: `sort: ["value1", "value2"]`
+
+The sorting order can be defined with `:asc` (ascending order, default, can be omitted) or `:desc` (for descending order).
+
+::: request Example request: Sorting on title by ascending order
+
+```graphql
+{
+  documents(sort: "title") {
+    data {
+      id
+    }
+  }
+}
+```
+
+:::
+
+::: request Example request: Sorting on title by descending order
+
+```graphql
+{
+  documents(sort: "title:desc") {
+    data {
+      id
+    }
+  }
+}
+```
+
+:::
+
+::: request Example request: Sorting on title by ascending order, then on price by descending order
+
+```graphql
+{
+  documents(sort: ["title:asc", "price:desc"]) {
+    data {
+      id
+    }
+  }
+}
+```
+
+:::
+
+## Pagination
+
+Queries can accept a `pagination` parameter. Results can be paginated either by page or by offset.
+
+:::note
+Pagination methods can not be mixed. Always use either `page` with `pageSize` **or** `start` with `limit`.
+:::
+
+### Pagination by page
+
+<!-- ? are we using a zero-based or a 1-based index? -->
+
+| Parameter              | Description | Default |
+| ---------------------- | ----------- | ------- |
+| `pagination[page]`     | Page number | 1       |
+| `pagination[pageSize]` | Page size   | 100     |
```

Comment:
Currently, the default is 10, it'll probably change before the stable though.
@alexandrebodin maybe you've got some default value in your mind?